### DISCUSSION TITLE
Markdown field custom height

### DISCUSSION THREAD

**Discussion by geowrgetudor** [filamentphp/filament#11509] [Ideas]
As far as I've dig there's no way to set a min & max height for the Markdown field. 

Seems like the min height is hard coded [here](https://github.com/filamentphp/filament/blob/af00a6b728a8b38d52a0d1c67b2f299aaa43eb1e/packages/forms/resources/js/components/markdown-editor.js#L114).

Doesn't it make sense to have it dynamic?

```
Markdown::make('body')
    ->minHeight(100)
    ->maxHeight(200)
```

Let me know what you think @danharrin

**Comment by danharrin** [comment]
Sure, that sounds fine

**Comment by beholdr** [comment]
Need same feature, created [PR](https://github.com/filamentphp/filament/pull/11695) that has been merged. Thanks!
